### PR TITLE
Fix demoshop link rendering in alerts

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: checkout/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ sidebar:
 of the checkout experience, integrating seamlessly into the merchant website
 through highly customizable and flexible components.
 
-Visit our [demoshop][demoshop]
+Visit our [demoshop](https://ecom.externalintegration.payex.com/pspdemoshop)
```
Comment (asbjornu) [inline checkout/index.md]: Can't you use `page.frontEndUrl` here?

File: payments/index.md [hunk 2]
```diff
@@ -157,10 +157,9 @@ After creating a payment, you can:
  All actions after creating the payment can be done by using our APIs, or from
  our admin tool. `abort` is only available when using APIs.
 
-Please visit our [demoshop][demoshop] to see our Payment Menu and Redirect
+Please visit our [demoshop](https://ecom.externalintegration.payex.com/pspdemoshop) to see our Payment Menu and Redirect
```
Comment (asbjornu) [inline payments/index.md]: As this link is not within an include, do we need to inline it?

File: checkout/index.md (hunk 1 above)
Comment (NullableInt) [reply checkout/index.md]: I can try

File: payments/index.md (hunk 2 above)
Comment (NullableInt) [reply payments/index.md]: I was a bit too aggressive in my changes.

**Pull request merged**